packages/web/src/components/switch/Switch.tsx

```diff
@@ -23,6 +23,7 @@ const Switch = ({
         checked={allowCheckedWhileDisabled ? isOn : isDisabled ? false : isOn}
         className={styles.inputCheckbox}
         type='checkbox'
+        onChange={_ => {}}
```

Comment:
this fixes a react error

Reply:
ah thank you for uncovering this bug! can we actually undo this change, and instead play a story to properly untangle the input/label onchange vs onclick issue? i just added it to my todo for tomorrow so ill add you to that pr?